packages/image/src/providers/cloudflare.js

```diff
@@ -0,0 +1,32 @@
+import { relative_url } from './utils';
+
+// (https://<ZONE>)/cdn-cgi/image/<OPTIONS>/<SOURCE-IMAGE>
+
+/** @type {import("types").GetURL<{ quality?: number, dpr?: number; fit?: string } & Record<string, any>>} */
+export function getURL({ src, width, options = {} }) {
```

Comment:
we should be able to use `height` here: https://developers.cloudflare.com/images/image-resizing/url-format/#height

or is that something you want to move to `options` since it's not universally supported (i.e. vercel) in which case we'd need to remove it from the types

Reply:
Yeah I think this is a leftover from some really early versions, probably best to just have `src` and `width` as universally supported